### PR TITLE
Seo 180844 ej2 mvc broken image

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Kindly review the broken image changes.

Regards,
Edith.

### CODE REVIEW

File: ej2-asp-core-mvc/grid/EJ2_ASP.MVC/selection/selection.md [hunk 1]
```diff
@@ -79,12 +79,12 @@ The grid supports three types of selection mode that can be set by using the [`M
...
-![mselection](images/mselection.jpg)  and then tap the desired rows or cells.
+![mselection in ASP.NET MVC Grid.](../../images/asp.net-mvc-grid-selection-mselection.jpg)  and then tap the desired rows or cells.
 
 N> Multi-selection requires the selection `Type` to be `Multiple`.
 
 The following screenshot represents a grid touch selection in the device.
 
-![Touch interaction](images/touch-selection.jpg)
\ No newline at end of file
+![Touch interaction in ASP.NET MVC Grid.](../../images/asp.net-mvc-grid-selection-touch-selection.jpg)
```
Comment (MallikaRK) [inline ej2-asp-core-mvc/grid/EJ2_ASP.MVC/selection/selection.md:90]: why double time selection word

File: ej2-asp-core-mvc/grid/EJ2_ASP.MVC/selection/selection.md [hunk 2]
```diff
@@ -79,12 +79,12 @@ The grid supports three types of selection mode that can be set by using the [`M
 ## Touch interaction
 
 When you tap a grid row on touchscreen device, the tapped row is selected. It also shows a popup 
-![selection](images/selection.jpg) 
+![selection in ASP.NET MVC Grid.](../../images/asp.net-mvc-grid-selection-popup.jpg) 
 for multi-row selection. To select multiple rows or cells, tap the popup
-![mselection](images/mselection.jpg)  and then tap the desired rows or cells.
+![mselection in ASP.NET MVC Grid.](../../images/asp.net-mvc-grid-selection-mselection.jpg)  and then tap the desired rows or cells.
```
Comment (MallikaRK) [inline ej2-asp-core-mvc/grid/EJ2_ASP.MVC/selection/selection.md:84]: give proper image name

File: ej2-asp-core-mvc/grid/EJ2_ASP.MVC/selection/selection.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: Selection in ##Platform_Name## Grid Component
-description: Learn here all about Selection in Syncfusion ##Platform_Name## Grid component of Syncfusion Essential JS 2 and more.
+title: Selection in ASP.NET MVC Grid Component | Syncfusion
```
Comment (MallikaRK) [inline ej2-asp-core-mvc/grid/EJ2_ASP.MVC/selection/selection.md:3]: Revert your changes in meta title and description

**Review by MallikaRK — COMMENTED** [review]
commented

File: ej2-asp-core-mvc/grid/EJ2_ASP.MVC/selection/selection.md (hunk 1 above)
Comment (Edithabich) [reply ej2-asp-core-mvc/grid/EJ2_ASP.MVC/selection/selection.md]: resolved

File: ej2-asp-core-mvc/grid/EJ2_ASP.MVC/selection/selection.md (hunk 2 above)
Comment (Edithabich) [reply ej2-asp-core-mvc/grid/EJ2_ASP.MVC/selection/selection.md]: updated

File: ej2-asp-core-mvc/grid/EJ2_ASP.MVC/selection/selection.md (hunk 3 above)
Comment (Edithabich) [reply ej2-asp-core-mvc/grid/EJ2_ASP.MVC/selection/selection.md:3]: changed due to CI error.

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**